rascaline/src/calculators/soap/radial_spectrum.rs

```diff
@@ -165,7 +165,7 @@ impl CalculatorBase for SoapRadialSpectrum {
 
     fn supports_gradient(&self, parameter: &str) -> bool {
         match parameter {
-            "positions" | "cell" => true,
+            "positions" | "cell" | "strain" => true,
```

Comment:
```suggestion
            "positions" | "strain" | "cell"  => true,
```

Reply:
this is fully internal, I don't think the order matters

Reply:
true it was just my consistency me that wanted to change this.